src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -58,9 +58,9 @@ void TestNetwork(ModelType& model,
 // network1 should be allocated with `new`, and trained on some data.
 template<typename MatType = arma::mat, typename ModelType>
 void CheckCopyFunction(ModelType* network1,
-                       MatType& trainData,
-                       MatType& trainLabels,
-                       const size_t maxEpochs)
+                        MatType& trainData,
```

Comment:
```suggestion
                       MatType& trainData,
```
Indentation issue here.